merlin-server/sql/merlin/tables/lifecycle_event.sql

```diff
@@ -0,0 +1,35 @@
+CREATE TYPE event_t AS ENUM ('begin', 'end', 'wait');
```

Comment:
I know the PR is essentially "pre-draft", but FWIW, most if not all of the other SQL files use lowercase keywords. (I've always found SQL kind of shouty, and I think good formatting tends to bring most of the benefits that capital keywords do anyway.)

Reply:
Good call! CHANGE HAS BEEN MADE